archeogitvsszz/utilities.py

```diff
@@ -10,3 +10,33 @@ class YAML:
     def read(path):
         with open(path, 'r') as file_:
             return yaml.safe_load(file_)
+
+
+class Calculation:
+    @staticmethod
+    def get_recall_and_precision(contributors, ground_truth):
+        true_positives = len(contributors & ground_truth)
+
+        false_negatives = 0
+        for contributor in ground_truth:
+            if contributor not in contributors:
+                false_negatives += 1
+
+        false_positives = 0
+        for contributor in contributors:
+            if contributor not in ground_truth:
+                false_positives += 1
```

Comment:
```suggestion
        false_positives = len(contributors - ground_truth)
```

False positive is a contributing commit which was incorrectly predicted by a blamer (i.e. it was falsely marked as positive). In other words, it is all commits in `contributors` that are _not in_ `ground_truth`.